app/lib/features/pins/actions/pin_update_actions.dart

```diff
@@ -115,6 +115,7 @@ Future<void> updatePinIcon(
     await updateBuilder.send();
     EasyLoading.dismiss();
     ref.invalidate(pinProvider);
+    ref.invalidate(pinListProvider);
```

Comment:
this only fixes the case where _we_ do the update. if the change comes from outside - another user - this will not trigger.

the proper fix for that entire situation is to switch the `pinList` to just return the `eventId`s and have each item be observed internally. But that is a bigger change not necessary now. We should write that up somewhere though.

Reply:
Adding TODO for now.

<img width="606" alt="Screenshot 2024-11-13 at 4 29 11 PM" src="https://github.com/user-attachments/assets/83d0bf19-d39f-4286-a126-cfae0904f269">